ocp-templates/README.md

```diff
@@ -1 +1,7 @@
-OCP Templates
+# OCP Templates & openshift scripts, for the global `CD` project
```

Comment:
I'd remove `for the global `CD` project`- as the scripts are not for it, and the templates are currently stored there but not used for it ...